lib/folio/eresource_holdings_builder.rb

```diff
@@ -85,7 +85,8 @@ def barcode(index)
     end
 
     def home_location_code
```

Comment:
Should we push this into `LocationsMap`, where we're already doing some of this location code massaging?

Reply:
How about into locations.tsv?

Reply:
I thought we were trying to keep our locations.tsv in sync with what libsys is using.